### ISSUE TITLE
RadAutocompleteTextView iOS native object is always undefined

### ISSUE THREAD

**Issue by Svettis2k** [telerik/nativescript-ui-feedback#285]
### Did you verify this is a real problem by searching [Stack Overflow](http://stackoverflow.com/questions/tagged/nativescript)?
Yes

### Tell us about the problem
When attempting to retrieve the native ios object from a RadAutocompleteTextViewComponent it is always returned as undefined. Using the same approach for android works as expected.

### Which platform(s) does your issue occur on?
iOS

### Please provide the following version numbers that your issue occurs with:
- **Progress NativeScript UI version:** 3.04
- **CLI:** 3.1.3
- **Cross-platform modules:** 3.1.1
- **Runtime(s):** 3.1.0

### Please tell us how to recreate the issue in as much detail as possible.
Run attached demo app, where onDidAutocomplete() in the HomeComponent illustrates the problem.
[test.zip](https://github.com/telerik/nativescript-ui-feedback/files/1272299/test.zip)

```typescript
onDidAutoComplete(args) {
        if (platform.isAndroid) {
            console.log(this.autocomplete.autoCompleteTextView.android); // Returns a com.telerik.widget.autocomplete.RadAutoCompleteTextView
            this.autocomplete.autoCompleteTextView.android.setText("NewText"); // Works as expected
        }
        if (platform.isIOS) {
            console.log(this.autocomplete.autoCompleteTextView.ios); // Returns undefined, but should return a TKRadAutoCompleteTextView
            this.autocomplete.autoCompleteTextView.ios.textField.text = "NewText"; // Fails with: ERROR TypeError: undefined is not an object (evaluating 'this.autocomplete.autoCompleteTextView.ios.textField')
        }
    }
```

**Comment by VladimirAmiorkov** [comment]
Hi @Svettis2k ,

Thank you for reporting this to us. We consider this a high priority and we have already implemented a fix for it and it will be available in our 3.1.0 version that is due to be available in couple of days.

**Comment by VladimirAmiorkov** [comment]
@Svettis2k As a workaround currently you can either access the native object by the `.nativeView` (this will return different object on the different platform) or access the private field `._ios` .
